README.md

```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Capstone Module 2
```

Comment:
You should have recorded a video presenting the features of the project.
The maximum length of the video is 5 minutes.
The video should contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
All the members in the video should appear in the video and talk about the project.
The video should contain a highlight about a piece of the code that you find interesting or are proud of making.

Reply:
Thank you @GzimAsani I do it as quick as possible

Reply:
- Hello there! the video link has been added. A gentle reminder, I'm the only participant in this video. Thank you very much for your understanding.